frontend/app/views/giraffe/thankyou.scala.html

```diff
@@ -26,6 +26,10 @@ <h1 class="support-thanks__title">Spread the word</h1>
             Enjoy more of the Guardian's journalism from breaking investigations to politics, sports, arts and more.
         </p>
 
+        <p class="support-thanks__message" data-shown="cad">
```

Comment:
There's quite a bit of duplication here.  It might be good to resolve this in a future PR.  Maybe externalise the copy to a messages file instead.